examples/with-cookie-auth-fauna/README.md

```diff
@@ -1,67 +0,0 @@
-# With Cookie Auth and Fauna
```

Comment:
We should not delete this, in case there are links to it. Let's add a link that points to the other example instead.